backend/src/schema/resolvers/reports.js

```diff
@@ -82,24 +84,24 @@ export default {
       const limit = params.first && typeof params.first === 'number' ? `LIMIT ${params.first}` : ''
 
       const reportReadTxPromise = session.readTransaction(async tx => {
-        const allReportsTransactionResponse = await tx.run(
-          `
-            MATCH (report:Report)-[:BELONGS_TO]->(resource)
-            WHERE (resource:User OR resource:Post OR resource:Comment)
-            ${filterClause}
-            WITH report, resource,
-            [(submitter:User)-[filed:FILED]->(report) |  filed {.*, submitter: properties(submitter)} ] as filed,
-            [(moderator:User)-[reviewed:REVIEWED]->(report) |  reviewed {.*, moderator: properties(moderator)} ] as reviewed,
-            [(resource)<-[:WROTE]-(author:User) | author {.*} ] as optionalAuthors,
-            [(resource)-[:COMMENTS]->(post:Post) | post {.*} ] as optionalCommentedPosts,
-            resource {.*, __typename: labels(resource)[0] } as resourceWithType
-            WITH report, optionalAuthors, optionalCommentedPosts, reviewed, filed,
-            resourceWithType {.*, post: optionalCommentedPosts[0], author: optionalAuthors[0] } as finalResource
-            RETURN report {.*, resource: finalResource, filed: filed, reviewed: reviewed }
-            ${orderByClause}
-            ${offset} ${limit}
-          `,
-        )
+        const cypher = `
+          MATCH (report:Report)-[:BELONGS_TO]->(resource)
+          WHERE (resource:User OR resource:Post OR resource:Comment)
+          ${filterClause}
+          WITH report, resource,
+          [(submitter:User)-[filed:FILED]->(report) |  filed {.*, submitter: properties(submitter)} ] as filed,
+          [(moderator:User)-[reviewed:REVIEWED]->(report) |  reviewed {.*, moderator: properties(moderator)} ] as reviewed,
+          [(resource)<-[:WROTE]-(author:User) | author {.*} ] as optionalAuthors,
+          [(resource)-[:COMMENTS]->(post:Post) | post {.*} ] as optionalCommentedPosts,
+          resource {.*, __typename: labels(resource)[0] } as resourceWithType
+          WITH report, optionalAuthors, optionalCommentedPosts, reviewed, filed,
+          resourceWithType {.*, post: optionalCommentedPosts[0], author: optionalAuthors[0] } as finalResource
+          RETURN report {.*, resource: finalResource, filed: filed, reviewed: reviewed }
+          ${orderByClause}
+          ${offset} ${limit}
+        `
+        const allReportsTransactionResponse = await tx.run(cypher)
+        log(allReportsTransactionResponse)
```

Comment:
Nah, it's a leftover of a previous commit. I wanted to log `cypher` and `params`. Then I discovered that the `allReportsTransactionResponse.summary` has both arguments as well. So just create a logging function for the response instead of loggin the two variables.

Reply:
ok, this `PR` has been including in the other https://github.com/Human-Connection/Human-Connection/pull/2470, correct?
If so, can we close this?